apps/kitchen-sink/src/components/common/ActionHeader/ActionHeader.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useRef } from "react";
+import "monday-ui-react-core/dist/main.css";
+import "./ActionHeader.scss";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faAngleLeft } from "@fortawesome/free-solid-svg-icons";
+import randomColorGenerator from "../../../utils/randomColorGenerator";
+
+const ActionHeader = ({ action, actionDescription, backgroundColor }) => {
+  const initialRandomColor = useRef(randomColorGenerator());
```

Comment:
Not sure why we need to have useRef here

Reply:
We use useRef here in order to generate random color when the component is rendering for the first time.
Any render after that won't change the color.